src/typings/electron-ipc.d.ts

```diff
@@ -23,6 +23,7 @@ type RecursivePartial<T> = {
 interface IpcMainEvents {
   'k8s-restart': () => void;
   'settings-read': () => void;
+  'images-namespaces-read': () => Array<string>;
```

Comment:
Please put this in the `imageEvents` region.

Reply:
done